lapis-docs/src/content/docs/concepts/response-format.mdx

```diff
@@ -11,7 +11,14 @@ A '200' response code indicates a successful request, while any other code signi
 Endpoints typically support returning data in JSON, [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) or
 [TSV](https://en.wikipedia.org/wiki/Tab-separated_values), with JSON being the default.
 Genomic sequences (such as those from `unalignedNucleotideSequences`, `alignedAminoAcidSequences`, etc.) are provided in the
-[FASTA format](https://en.wikipedia.org/wiki/FASTA_format).
+[FASTA format](https://en.wikipedia.org/wiki/FASTA_format) by default,
+but can also be requested in JSON or [NDJSON](https://github.com/ndjson/ndjson-spec) format.
+
+:::note
+NDJSON is in particular useful for downloading large datasets, as it allows for streaming data processing.
+You can read line by line without loading the entire file into memory.
+Since every line is a valid JSON object, it is usually easier to handle programmatically than FASTA.
+:::
 
 ## Example
 
```

Comment:
I think we need to adapt the pictures for the example. The data format is no longer required, but the reportTo is.

![grafik](https://github.com/user-attachments/assets/a4753384-aa34-4c84-a1f7-2da4adfa651d)

Reply:
I updated it :+1: